pyomo/contrib/cp/repn/docplex_writer.py

```diff
@@ -667,6 +668,12 @@ def _get_bool_valued_expr(arg):
 
 
 def _handle_monomial_expr(visitor, node, arg1, arg2):
+    # Monomial terms show up a lot.  This handles some common
+    # simplifications (necessary in part for the unit tests)
+    if arg2[1].__class__ in EXPR.native_types:
+        return _GENERAL, arg1[1] * arg2[1]
+    elif arg1[1] == 1:
+        return arg2
```

Comment:
`arg2` is already the 2-tuple of (type, value).  Since `1*` is identity, we don't have to look at / update / care what the type is for arg2 and just pass it up untouched.